tests/backend/test_postgres_db.py

```diff
@@ -0,0 +1,62 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import re
+
+import pytest
+
+from parsec.backend.config import PostgreSQLBlockStoreConfig
+
+
+@pytest.mark.trio
+@pytest.mark.postgresql
+async def test_postgresql_connection_ok(
+    postgresql_url, alice, bob, backend_factory, backend_sock_factory
+):
+    async with backend_factory(
+        config={"blockstore_config": PostgreSQLBlockStoreConfig(), "db_url": postgresql_url}
+    ):
+        pass
+
+
+@pytest.mark.trio
+@pytest.mark.postgresql
+async def test_postgresql_connection_not_ok(
+    postgresql_url, alice, bob, backend_factory, backend_sock_factory, caplog
+):
+    postgresql_url = re.sub(
+        ":(\\d+)/", lambda m: f":{str(int(m.group()[1:-1])+1)}/", postgresql_url
```

Comment:
This is too complex for it own good ^^

Besides we should use `unused_tcp_port` fixture to ensure the address is invalid
```python
bad_postgresql_url = f"postgresql://localhost:{unused_tcp_port}/dummy"
```

Reply:
Ofc...